### PR TITLE
code_review_25_02

### PR DESCRIPTION
**Pull request by ruslan-hontar**
Загалом. Молодець. Видно, що засвоїв попередні зауваження.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,24 +3,43 @@
...
+    "eslint-config-prettier": "^6.10.0",
+    "eslint-plugin-prettier": "^3.1.2",
+    "eslint-plugin-react": "^7.18.3",
+    "google-map-react": "^1.1.6",
+    "prettier": "^1.19.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+    "react-router-dom": "^5.1.2",
+    "react-router-hash-link": "^1.2.2",
     "react-scripts": "3.3.1",
```
Comment (ruslan-hontar) [inline package.json]: Got problem with `"react-scripts": "3.3.1",`. Update library to latest `3.4.0` fixed my issue

File: src/App/components/Footer/index.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        </div>
+      </div>
+      <hr />
+      <div className={classes.row}>
+        <div>
+          <Link to="/#start">
+            <div className={classes.logo}>
+              <div>
+                <img src={logo} alt="" />
+              </div>
```
Comment (ruslan-hontar) [inline src/App/components/Footer/index.tsx]: Don't  use not needed tags `<div> ... </div>`

File: src/App/components/Post/index.tsx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import { HashLink as Link } from 'react-router-hash-link'
+
+import dafaultClasses from './Post.module.scss' //TODO
+
+interface IProps {
+  img?: string
+  title: string
+  body: string
+  author: string
+  date: string //TODO Date
```
Comment (ruslan-hontar) [inline src/App/components/Post/index.tsx]: Great idea to use `TODO:`'s. Take a look to this extension for VC code `https://travis-ci.org/Gruntfuggly/todo-tree`

File: src/App/components/Post/index.tsx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  timeToRead: number
+  classes?: typeof dafaultClasses
+}
+
+const post = (props: IProps) => {
+  const classes = props.classes
+    ? { ...props.classes, ...dafaultClasses }
+    : dafaultClasses
```
Comment (ruslan-hontar) [inline src/App/components/Post/index.tsx]: Also you can use array of the classes : `{[ props.classes, ... ]}

File: src/App/components/Toolbar/index.tsx [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+class HeaderMain extends Component {
+  state = {
+    opened: false,
+  }
+
+  toggleDrawer = () => {
+    this.setState({
+      opened: !this.state.opened,
+    })
+  }
```
Comment (ruslan-hontar) [inline src/App/components/Toolbar/index.tsx]: You can use hooks for avoid to make class Component. More information you can take from this link `https://uk.reactjs.org/docs/hooks-intro.html`

File: src/App/screens/Homepage/OurClients/index.tsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do
+            eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+            ad minim veniam.
+          </p>
+        </div>
+        <div className={classes.clientsList}>
+          {CLIENTS_LOGO.map((logo, index) => (
+            <Link to="#" key={index}>
+              <div>{logo}</div>
+            </Link>
```
Comment (ruslan-hontar) [inline src/App/screens/Homepage/OurClients/index.tsx]:
```
{CLIENTS_LOGO.map((logo, index) => (
```
to
```
{CLIENTS_LOGO.map((logo: string, index:number) => (
```

File: package.json (hunk 1 above)
Comment (MykolaRomaniv) [reply package.json]: fixed

File: src/App/components/Post/index.tsx (hunk 3 above)
Comment (MykolaRomaniv) [reply src/App/components/Post/index.tsx]: i`m already using it 🙂

**Pull request merged**